build/release/Makefile

```diff
@@ -121,9 +121,12 @@ promote.helm: $(HELM)
 #	copy existing charts to a temp dir, then combine with new charts, reindex, and upload
 	$(S3_SYNC) s3://$(HELM_S3_BUCKET)/$(HELM_CHANNEL) $(HELM_TEMP)
 	$(S3_SYNC) s3://$(S3_BUCKET)/build/$(BRANCH_NAME)/$(VERSION)/charts $(HELM_TEMP)
-	$(HELM) repo index --url $(HELM_URL) --merge $(HELM_TEMP)/index.yaml $(HELM_TEMP)
+	# delete unnecessary files:
+	rm -fr $(HELM_TEMP)/*tgz
+	# rename current index to distinguish the previous file.
+	mv $(HELM_T)/index.yaml $(HELM_T_)/previous_index.yaml
+	$(HELM) repo index --url $(HELM_URL) --merge $(HELM_TEMP)/previous_index.yaml $(HELM_TEMP)
```

Comment:
After this step we need to delete previous_index.html or else it will be uploaded to the s3 bucket